tests/functional/broadcast/00-simple/flow.cylc

```diff
@@ -92,7 +92,9 @@
            sed -i '/BASH_XTRACEFD/d' ${PREPLOG}.err
 
            diff -u "${CYLC_WORKFLOW_RUN_DIR}/expected-prep.out" ${PREPLOG}.out
-           diff -u "${CYLC_WORKFLOW_RUN_DIR}/expected-prep.err" ${PREPLOG}.err
+           # Let's not compare stderr here; it can be contaminated by debug
+           # output from parsec (repeated config items).
+           # diff -u "${CYLC_WORKFLOW_RUN_DIR}/expected-prep.err" ${PREPLOG}.err
```

Comment:
Remove the `--debug` and the problem goes away.

Reply:
> Remove the --debug and the problem goes away.

And so does the triggering info, so the reference test fails.

I figured there should not be any problem with the stderr file if the stdout is 100% correct. (However, I didn't check the details).

Reply:
(merged your log-level change, so this is moot point now anyway!)